vignettes/DataAnalysis.Rmd

```diff
@@ -164,41 +170,44 @@ The default flagging function for the normal approximation method is `Flag_Norma
+-   `GroupLevel`: Type of Group specified in `GroupID` (Country, Site, QTL)
 -   `Numerator`: Cumulative Number of Event(s) of Interest (in this example, number of AEs reported across subjects); Carried from `dfAnalyzed`
 -   `Denominator`: Cumulative Time on Treatment (in days, across subjects); Carried from `dfAnalyzed`
 -   `Metric`: Rate of Event(s) of Interest (in this example, number of AEs reported over the course of treatment in days)
 -   `Score`: Calculated Residual (per site)
 -   `Flag`: Flag Indicating Possible Statistical Outliers; Valid values for this variable include -2, -1, 0, 1, and 2, which determine the "extremeness" of the outlier. -2 and 2 represent more extreme outliers, -1 and 1 represent less extreme outliers, and 0 represents a non-outlier.
 
-```{r}
-# dfSummary <- Summarize(dfFlagged)
+```{r include = TRUE}
+dfSummary <- Summarize(dfFlagged)
 ```
 
 ```{r, echo = FALSE}
-# dt(dfSummary)
+dt(dfSummary)
 ```
 
 ------------------------------------------------------------------------
 
 ### 6. Data Visualization
 
-Data visualizations are built into each `Assess()` function/KRI. For a Poisson AE assessment, a scatter plot of Total Exposure (in days, on log scale) vs Total Number of Event(s) of Interest (on linear scale) is created. Each data point represents one site with outliers displayed in yellow or red, depending on the "extremeness" of the KRI value. The `Visualize_Scatter()` function takes inputs of `dfTransformed` to define bounds and `dfFlagged` to plot the data. Using our example AE data, we see the following scatter plot:
+For a Normal Approximation AE assessment, a scatter plot of Total Exposure (in days, on log scale) vs Total Number of Event(s) of Interest (on linear scale) is created. Each data point represents one site with outliers displayed in yellow or red, depending on the "extremeness" of the KRI value. The `Visualize_Scatter()` function takes inputs of `dfBounds`-- which defines bounds based on statistical model, `dfTransformed` and thresholds using `Analyze_NormalApprox_PredictBounds()`-- and `dfFlagged` to plot the data. Using our example AE data, we see the following scatter plot:
 
-```{r}
-# dfBounds <- Analyze_NormalApprox_PredictBounds(dfTransformed, vThreshold = c(-3, -2, 2, 3))
+```{r include = TRUE}
+dfBounds <- Analyze_NormalApprox_PredictBounds(dfTransformed, vThreshold = c(-3, -2, 2, 3))
 
-# chart <- Visualize_Scatter(dfFlagged, dfBounds)
+chart <- Visualize_Scatter(dfFlagged, dfBounds)
 ```
 
 ```{r, echo = FALSE, fig.width=7, fig.height=5}
-# chart
+chart
 ```
 
+A full explanation of all Data Visualizations in the `{gsm}` package is outlined in the Reporting Workflow vignette (ADD LINK).
+
 ------------------------------------------------------------------------
 
 # Recap - Normal Approximation Adverse Event KRI
 
--   `dfInput` used as original input using `AE_Map_Raw()`
+-   `dfInput` used as original input using `InputRate()`
```

Comment:
```suggestion
-   `dfInput` used as original input using `Input_Rate()`
```

Reply:
On row 215 below, we reverse the pattern and list the function (`Visualize_Scatter()`) first, and the data it takes second. The rest list the data first, so it would make more sense to maintain the pattern. (I can't comme t on that row directly since it hasn't changed).